sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -513,7 +545,7 @@ def get_sdk_package_name():
   """For internal use only; no backwards-compatibility guarantees.
 
   Returns the PyPI package name to be staged to Google Cloud Dataflow."""
-  container_version = get_required_container_version()
+  container_version = _get_required_container_version()
```

Comment:
There is a bug here, and not directly related to this PR.

We should select package based on what packages are installed, not from the container version. Similar to `get_sdk_name_and_version`. In order to avoid another `try/catch` this code could be changed to

```
sdk_name, _ = get_sdk_name_and_version
if sdk_name == 'Google Cloud Dataflow SDK for Python':
  return GOOGLE_PACKAGE_NAME
else:
  return BEAM_PACKAGE_NAME
```

If easy you can do this in the same PR. If not I will do a follow up PR.